model/CoreLN.ts

```diff
@@ -30,7 +30,7 @@ export type ListFunds = {
 export type Channel = {
   peer_id: string;
   our_amount_msat: string | number;
-  amount_msat: string | number;
```

Comment:
this can be also a number, because core lightning is migrating from string to number! we need to keep this in mind